docs/pages/versions/unversioned/index.mdx

```diff
@@ -127,6 +127,6 @@ Each version of Expo SDK supports a minimum OS version of Android and iOS. For A
 | ---------------- | --------------- | ------------------- | ----------- | ------------- |
 | 53.0.0           | 7+              | 35                  | 15.1+       | 16.0+         |
 | 52.0.0           | 7+              | 35                  | 15.1+       | 16.0+         |
-| 51.0.0           | 6+              | 34                  | 13.4+       | 15.4          |
+| 51.0.0           | 6+              | 34                  | 13.4+       | 15.4 - 16.2   |
```

Comment:
@kbrandwijk, can you also update the SDK 53 reference index page: https://docs.expo.dev/versions/latest/#support-for-android-and-ios-versions since we also mention SDK 51 in the table there?

Reply:
And probably 51 and 52 as well, as they all have that same table... Will update

Reply:
For latest, it's in the 53 folder, right? If so, I think I got them all.

Reply:
Thank you!